template/helper/player.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @package    DOCman
+ * @copyright   Copyright (C) 2011 - 2014 Timble CVBA (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        http://www.joomlatools.com
+ */
+
+class ComFilesTemplateHelperPlayer extends KTemplateHelperAbstract
+{
+    protected static $HTML5_VIDEO_FORMATS = array('mp4', 'webm', 'ogg');
+
+    protected static $HTML5_AUDIO_FORMATS = array('aac', 'mp3', 'ogg', 'flac','x-flac', 'wave', 'wav', 'x-wav', 'x-pn-wav');
+
+    protected static $declared = false;
+
+    /*
+    public function load()
```

Comment:
The reason I kept that was because @ercanozkaya suggested that I insert that output in layouts. However that approach doesn't work when we need to use the template in the content plugin, so instead I created a singleton that imports the assets only once. I left the commented out code so Ercan would know about the change. I was going to clean it up afterwards.